frontend/src/components/pages/CalibrationPage.tsx

```
@@ -0,0 +1,707 @@
+import React, { useState, useEffect } from 'react';
+import Plot from 'react-plotly.js';
+import { 
+  calibrationApi, 
+  CalibrationResult, 
+  CalibrationRequest, 
+  PerformanceAnalysis,
+  SurfaceQualityMetrics,
+  CalibrationMetrics 
+} from '../../services/calibrationApi';
+
+interface CalibrationPageProps {}
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)